docs/samples/Microsoft.ML.Samples/Dynamic/WordEmbeddingTransform.cs

```diff
@@ -0,0 +1,73 @@
+﻿using System;
+using Microsoft.ML.Data;
+using Microsoft.ML.Transforms.Text;
+namespace Microsoft.ML.Samples.Dynamic
+{
+    class WordEmbeddingTransform
+    {
+        public static void ExtractEmbeddings()
+        {
+            // Create a new ML context, for ML.NET operations. It can be used for exception tracking and logging, 
+            // as well as the source of randomness.
+            var ml = new MLContext();
+
+            // Get a small dataset as an IEnumerable and convert to IDataView.
+            var data = SamplesUtils.DatasetUtils.GetSentimentData();
+            var trainData = ml.Data.ReadFromEnumerable(data);
+
+            // Preview of the data.
+            //
+            // Sentiment    SentimentText
+            // true         Best game I've ever played.
+            // false        ==RUDE== Dude, 2.
+            // true          Until the next game, this is the best Xbox game!
+
+            // Pipeline which goes through SentimentText and normalize it, tokenize it by words, and removes default stopwords.
+            // After all that cleaning, we apply WordEmbedding with GloVeTwitter25D.
+            // 25D means each word mapped into 25 dimensional space, basically each word represented by 25 float values.
+            var wordsPipeline = ml.Transforms.Text.NormalizeText("NormalizedText", "SentimentText")
+                .Append(ml.Transforms.Text.TokenizeWords("Words", "NormalizedText"))
+                .Append(ml.Transforms.Text.RemoveDefaultStopWords("CleanWords", "Words"))
+                .Append(ml.Transforms.Text.ExtractWordEmbeddings("WordEmbeddings", "CleanWords", WordEmbeddingsExtractingTransformer.PretrainedModelKind.GloVeTwitter25D));
```

Comment:
Might want to add a comment encouraging the user to try other models. The FastText300D is most often the winner for accuracy (though is larger/slower).

Reply:
It's not about best practices, it's about how it works.

---
In reply to: [252511336](https://github.com/dotnet/machinelearning/pull/2339#discussion_r252511336) [](ancestors = 252511336)